subr/check_version.sh

```diff
@@ -0,0 +1,8 @@
+export selfupdate_macports="false"
+export macports_version="$1"
+
+if [ "$1" = "" ]; then
```

Comment:
The empty string does not really convey the idea that we are going to use the latest available version. What about using the string `LATEST` instead ?